owncloudComLibrary/src/main/java/com/owncloud/android/lib/common/http/methods/HttpBaseMethod.kt

```diff
@@ -99,6 +100,11 @@ abstract class HttpBaseMethod constructor(url: URL) {
         return response.body?.byteStream()
     }
 
+    /**
+     * returns the final url after following the last redirect.
+     */
+    open fun getFinalUrl() = response.request.url
```

Comment:
one can use this function to fix:
https://github.com/owncloud/android/blob/56a0c79a389ee7a5c1fe7a8d8f78c9f1a153618b/owncloudData/src/main/java/com/owncloud/android/data/authentication/datasources/implementation/OCRemoteAuthenticationDataSource.kt#L53

Reply:
Not available from an operation scope. Let's check other ways...